app/config.js

```diff
@@ -6,12 +6,13 @@ const config = {
     gitRevision: process.env.GIT_REVISION,
     frontendPublicHttpProtocol: process.env.PUBLIC_PROTOCOL || 'http',
     featureToggles: {
-        url: process.env.FEATURE_TOGGLES_API_URL || 'http://localhost:8282',
+        url: process.env.FEATURE_TOGGLES_API_URL || 'http://localhost:9999',
```

Comment:
We should probably use a more 'real' port num : 8292? Make sure it wont clash

Reply:
Changed